content/webapp/components/IIIFViewer/ViewerStructures.tsx

```diff
@@ -74,15 +72,17 @@ const Structures: FunctionComponent<Props> = ({
   work,
   query,
 }) => {
-  return ranges.length > 0 ? (
+  return ranges && ranges.length > 0 ? (
```

Comment:
Is optional chaining (`ranges?.length > 0`) no good here for TypeScript reasons or something?

Reply:
yeah, it complains ranges.length could be undefined